src/transmission/bidib_transmission_receive.c

```diff
@@ -408,18 +413,20 @@ void bidib_handle_received_message(uint8_t *message, uint8_t type,
 			                           message, action_id);
 			dcc_address.addrl = message[data_index];
 			dcc_address.addrh = message[data_index + 1];
-			pthread_rwlock_wrlock(&bidib_state_track_rwlock);
+			//pthread_rwlock_wrlock(&bidib_state_track_rwlock);
+			// Both for bidib_state_cs_accessory_manual
+			pthread_mutex_lock(&trackstate_accessories_mutex);
 			pthread_rwlock_rdlock(&bidib_state_boards_rwlock);
-			bidib_state_cs_accessory_manual(node_address, dcc_address,
-			                                message[data_index + 2]);
+			bidib_state_cs_accessory_manual(node_address, dcc_address, message[data_index + 2]);
 			pthread_rwlock_unlock(&bidib_state_boards_rwlock);
-			pthread_rwlock_unlock(&bidib_state_track_rwlock);
+			//pthread_rwlock_unlock(&bidib_state_track_rwlock);
+			pthread_mutex_unlock(&trackstate_accessories_mutex);
 			free(message);
 			break;
 		case MSG_LC_STAT:
 			// update state
-			bidib_log_received_message(addr_stack, seqnum, type, LOG_DEBUG,
-			                           message, action_id);
+			//bidib_log_received_message(addr_stack, seqnum, type, LOG_DEBUG,
+			//                           message, action_id);
```

Comment:
Delete lines

Reply:
Delete? Then this'd be the only message where we dont log it that way.